include/ddc/kernels/splines/bsplines_non_uniform.hpp

```diff
@@ -75,23 +85,20 @@ class NonUniformBSplines : detail::NonUniformBSplinesBase
         int m_nknots;
 
     public:
+        /// @brief The type of discrete dimension representing the BSplines.
         using discrete_dimension_type = NonUniformBSplines;
 
+        /// @brief The type of discrete domain identifying the BSplines.
         using discrete_domain_type = DiscreteDomain<DDim>;
```

Comment:
Does a domain identify an object?

Reply:
`discrete domain whose elements identify the BSplines` ?

Reply:
I'm not sure. Maybe, this works if DiscreteDomain is properly documented :sweat_smile: but as the aim for docs is to make it easily readable for someone who is unfamiliar with DDC I think it would be better if there was also more information here.

By the way, do you mind not closing conversations that I start (auto-close because you applied the suggestion as-is is fine). I like to go back and compare my comment to the commits so I don't have to redo a complete review

Reply:
:+1: perfect thanks

Reply:
I think as we are inside DDC, we can assume the user is aware of what a DiscreteDomain is. So to me:

```
The type of a DiscreteDomain whose elements identify the B-splines.
```

looks ok.